apps/dashboard/app/views/projects/_job_details.turbo_stream.erb

```diff
@@ -5,8 +5,25 @@
 
 <turbo-stream action="replace" target="<%= id %>">
   <template>
-    <div>
-      <%= job.id %> is currently <%= job.status.to_s %>.
+    <button class="badge <%= status_class(job.status.to_s) %> rounded-pill border-0 btn" type="button" data-bs-toggle="collapse" data-bs-target="#<%= id %>Data">
+      <span>
+        <%= job.id %> <%= status_text(job.status.to_s) %>
+      </span>
+    </button>
+    <div class="collapse" id="<%= id %>Data">
+      <div class="card card-body">
+        <% readable_job_data(job).each do |category_name, category| %>
+          <table class="table table-bordered table-sm m-0 mb-2">
+            <thead><%= category_name %></thead>
+            <% category.each do |name, value| %>
+              <tr>
+                <td><strong><%= name %></strong></td>
+                <td><%= value %></td>
+              </tr>
+            <% end %>
+          <% end %>
+        </table>
```

Comment:
OK - I think I've got this now. I think this should just be 1 table instead of 3. I think that'll simplify the computation for this object as well.